draft-ietf-ppm-dap.md

```diff
@@ -1723,6 +1738,12 @@ the aggregate share was used in a batch. This is due to the requirement that the
 batch interval respect the boundaries defined by the DAP parameters. (See
 {{batch-validation}}.)
 
+Furthermore, the aggregators must store data related to a task as long as current epoch
+time has not passed this task's `max_task_lifetime`. Aggregator MAY delete the task and all
+data pertaining to this task after `max_task_lifetime`. Implementors SHOULD provide for some
+leeway so collector can collect the batch after some delay.
```

Comment:
We already define type `Time` in terms of the Unix epoch, so no need to refer to that here.
```suggestion
Furthermore, the aggregators must store data related to a task as long as the current time
has not passed this task's `max_task_lifetime`. Aggregator MAY delete the task and all
data pertaining to this task after `max_task_lifetime`. Implementors SHOULD provide for some
leeway so collector can collect the batch after some delay.
```